test/unit/rules/quotes-test.js

```diff
@@ -33,9 +33,23 @@ generateRuleTests({
   ],
 
   bad: [
+    {
+      config: 'double',
+      template: "{{component 'one {{thing}} two'}}",
+      fixedTemplate: '{{component "one {{thing}} two"}}',
+
+      result: {
```

Comment:
Can we use `verifyResults` instead so we can have a convenient snapshot?

Reply:
done